.github/workflows/xfstests.yaml

```diff
@@ -3,14 +3,15 @@ name: xfstests
 on:
   push:
     branches:
-      - '**'
+      - 'xfstestsci**'
     paths:
       - 'fs/xfstests.py'
       - 'fs/xfstests.py.data/*/ci.yaml'
       - '.github/workflows/xfstests.yaml'
   pull_request:
     branches:
-      - '**'
+      - 'master'
+      - 'xfstestsci**'
```

Comment:
can we have pull requests restricted to `master` only?

Reply:
Sure.